plugins/Stk/Mallets/Mallets.cpp

```diff
@@ -293,6 +296,29 @@ void MalletsInstrument::playNote( NotePlayHandle * _n,
 			m_isOldVersionModel.value() ? 100.0 : 200.0;
 		const float vel = _n->getVolume() / velocityAdjust;
 
+		const float random = m_randomModel.value();
+		float hardness = m_hardnessModel.value();
+		float position = m_positionModel.value();
+		float modulator = m_modulatorModel.value();
+		float crossfade = m_crossfadeModel.value();
+
+		if(p < 9)
+		{
+			hardness += random * static_cast<float>(fast_rand() % 128) - 64.0;
+			hardness = std::clamp(0.0f, hardness, 128.0f);
```

Comment:
Unlike `qBound`, `std::clamp` takes its arguments in the order value, minimum, maximum.

Reply:
A non trivial error I'd say. Thanks!